lib/Raven/Client.php

```diff
@@ -363,6 +363,7 @@ public function __destruct()
     {
         // Force close curl resource
         $this->close_curl_resource();
+        $this->force_send_async_curl_events();
```

Comment:
We need to send all events before dying. I saw this problem while I was writing this PR.